electrumsv/gui/qt/network_dialog.py

```diff
@@ -1277,6 +1281,17 @@ def _on_menu_delete_server(self, entry: ServerListEntry) -> None:
                 self):
             return
 
+        def done_callback(future: concurrent.futures.Future[list[NetworkServerRow]]) -> None:
+            # Skip if the operation was cancelled.
+            if future.cancelled():
+                return
+            try:
+                future.result()
+            except DatabaseUpdateError as e:
+                self._main_window_weakref.show_error(e.args[0])
+
+            self.server_disconnected_signal.emit()
```

Comment:
Since you are fixing other things, please rename this event which is not used by anything else and obviously has nothing to do with disconnection to "refresh signal" or something.